src/scenes/Moves/ducks.js

```diff
@@ -59,6 +59,8 @@ export const lastMoveIsCanceled = state => get(state, 'moves.latestMove.status')
 
 export const selectedMoveType = state => get(state, 'moves.currentMove.selected_move_type');
 
+export const isPpm = state => get(state, 'ppm.currentPpm', false);
```

Comment:
you might want wrap this in a bool
```suggestion
export const isPpm = state => Boolean(get(state, 'ppm.currentPpm', false));
```

Reply:
added